common/config/babel/.babelrc.js

```diff
@@ -94,6 +92,8 @@ process.env['COMMUNICATION_REACT_FLAVOR'] !== 'beta' &&
         // Demo feature. Used in live-documentation of conditional compilation.
         // Do not use in production code.
         'stabilizedDemo',
+        // Pinned Participants
+        'pinned-participants'
```

Comment:
Do we remove this from here along with all the conditional compilation flags?

Reply:
I think once stabilized we can remove the CC from pinned layouts